backend/src/services/auth.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Awesome!! Creating a service for auth and firebase is a great idea. These organizational things might seem not important right now but as our codebase increases in size, its usefulness will become clearer.